great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -139,6 +139,10 @@ def __init__(
 
         self._rule_states = []
 
+    @property
+    def ge_cloud_id(self) -> Optional[str]:
+        return self._id
```

Comment:
Named this property `ge_cloud_id` just for parity but I'm happy to change it to whatever we deem acceptable.